tests/foreman/cli/test_variables.py

```diff
@@ -157,15 +204,17 @@ def test_negative_create(self):
 
         @steps:
 
-        1. Create a smart Variable with Invalid name and valid default value.
+        1. Create a smart Variable with Invalid name.
 
         @assert: The smart Variable is not created.
-
-        @caseautomation: notautomated
         """
+        for name in invalid_values_list():
+            with self.subTest(name):
+                with self.assertRaises(CLIReturnCodeError):
```

Comment:
We can choose style whatever we like according to different discussion that we have in irc. That new style must be used in case we have over indentation in the block itself, but otherwise old style is more preferable due more readability